vm/devices/get/underhill_config/src/lib.rs

```diff
@@ -355,6 +355,9 @@ pub enum Vtl2SettingsErrorCode {
     NetworkingAddNicFailed => (Network, Configuration),
     /// Failed to remove NIC
     NetworkingRemoveNicFailed => (Network, Configuration),
+
+    /// Failed to update VTL2 servicing options
+    Vtl2ServicingOptions => (Underhill, Configuration),
```

Comment:
looks like this is unused in the latest revision of the PR? at least - based on a quick grep of the codebase

Reply:
You're right, wondering why clippy didn't flag that.

Reply:
this is exposed as `pub` in this crate, and clippy / rust don't do dead-code analysis between crates.

maybe someday we'll have CodeQL rules that will let us catch these sorts of things across the project...